### PR TITLE
Preparing for release 1.0.0

### PR DESCRIPTION
**Pull request by eapearson**
This PR is for any final work for release 1.0.0.

Includes:

- update CHANGELOG

### CODE REVIEW

**Comment by eapearson** [comment]
Updated integration tests so that they pass with user kbaseuitest. These tests are not invoked by GHA, but should be run by a dev locally as if they were.

File: tests/integration/data/legacy/case-07.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# About case 7
```
Comment (scanon) [inline tests/integration/data/legacy/case-07.md:1]: Do you want to say more?

File: tests/integration/data/legacy/case-08.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Case 8 Description
```
Comment (scanon) [inline tests/integration/data/legacy/case-08.md:1]: Search with sorting?

**Review by scanon — CHANGES_REQUESTED** [review]
Since the markdown is there, maybe add better descriptions for those few examples.  Otherwise LGTM

File: tests/integration/data/legacy/case-07.md (hunk 1 above)
Comment (eapearson) [reply tests/integration/data/legacy/case-07.md:1]: I was copying tests from a python file to json files, and transferred comments to a companion markdown file. Not all of the json-in-python had comments. I'll inspect those tests and add relevant comments.

File: tests/integration/data/legacy/case-08.md (hunk 2 above)
Comment (eapearson) [reply tests/integration/data/legacy/case-08.md:1]: As for the other comment, I just left placeholders when there were no existing test description. I'll add the appropriate description.

**Pull request merged**